content/md/imports-exports/tailored-export.md

```diff
@@ -0,0 +1,86 @@
+
+# Create a tailored product export profile
+
+To create a product export:
+1. Go to `Exports`
+1. Click on `Create export profile`
+1. Type a unique export code and a label, then select the job `Tailored product export` or `Tailored product model export`. You can select either CSV or XLSX for each job.
+![Create a new product export profile](../img/TailoredExport_Create-export-profile.png)
+1. Save your new export profile
+
+# Edit a tailored product export profile
+
+Once you've created your tailored export profile, or if you want to edit an existing tailored product export profile:
+1. Go to `Exports`
+1. Click on the tailored export profile you want to edit
+1. Click on `Edit` located in the top right corner of the screen  
+
+## Discover the different tabs of this profile
+
+As in all export profiles, the tailored export profile displays different tabs where you can configure it. Here's the list of the different tabs:
+
+* `Properties`: Define the label of your export profile
+* `Permissions`: Set permissions to allow specific user groups to execute or edit your export profile (Enterprise Edition only).
+* `Global settings`: Define the max number of lines per file you want, the field delimiter (CSV format only) and the field enclosure character (CSV format only).
+* `Product selection`: Use the provided filters to decide which products will be exported (see [Focus on "Product selection" section](tailored-export.html#focus-on-product-selection-section)").
+* `Export structure`: Build your export structure (see [Focus on "Export structure" section](tailored-export.html#focus-on-export-structure)").
+* `History`: Display the modifications made by any user on your export profile.
+
+
+### Focus on 'Product selection' section
+
+In this tab, you can select the products which will be exported.
+A certain number of filters are available, helping you to refine your product selection, such as family, category, status, completeness, data quality or even insert a list of identifiers. You can also add new attributes to use them to filter your products.
+
+::: tips
+Note that you can now select any category from any category tree available in your PIM, including categories that aren't linked to a channel.
+:::
+
+![Select multiple category trees](../img/TailoredExport_Select-categories.png)
+
+### Focus on 'Export structure'
+
+In this tab, you can build the structure of the file.
+You need to create the columns of your file, name them & link them to your PIM attributes in order for your export profile to be ready to be launched.
+To do so, please follow the steps below:
+1.  Add a first column by clicking on `Add first column`.
+![Add first column](../img/TailoredExport_Add-first-column.png)  
+::: tips
+If you want to add columns in bulk from an Excel file, you can copy all the columns in your Excel and paste them in the last column created, so that it creates automatically all your columns in your export.
```

Comment:
```suggestion
If you want to add columns in bulk from an Excel file, you can copy all the columns header in your Excel and paste them in the last column created, so that it creates automatically all your columns in your export.
```